qemu/tests/block_resize.py

```diff
@@ -81,8 +81,8 @@ def check_shrink_completion(vol_id, size):
         def _check_event_cmd(cmd):
             status, output = session.cmd_status_output(cmd)
             test.log.debug("Get event: %s" % output)
-            test.log.debug("Expect contain msg: %s" % msg)
-            return status == 0 and regexp.search(output)
+            test.log.debug("Expect contain msg: %s" % msg)  # pylint: disable=E0606
```

Comment:
hi @luckyh , do we want to change the usage of the test.log.xxx, as test.log.debug("Expect contain msg: %s", msg)?

Reply:
@zhencliu good catch, and I remember that this issue can be caught by pylint/inspektor as well, however at the moment we just disabled that. I'd suggest filing another issue for both enabling the check and fixing it over the tp-qemu repo at the same time, thanks!